aws/ecr/ecr-repo-app.tf

```diff
@@ -0,0 +1,40 @@
+module "kops_ecr_app" {
+  source       = "git::https://github.com/cloudposse/terraform-aws-kops-ecr.git?ref=tags/0.1.0"
+  namespace    = "${var.namespace}"
+  stage        = "${var.stage}"
+  name         = "${element(repositories_names,0)}"
```

Comment:
This is an invalid variable :P

Reply:
Also, I don't understand why it's valid that we create an ECR for only the first (`0`) repository.